### DISCUSSION TITLE
What is the best plan to operate 20 viewers?

### DISCUSSION THREAD

**Discussion by newworld00** [KevinBytesTheDust/CVAmp#212] [General]
What is the best plan to operate 20 viewers?thx

**Comment by jlplenio** [comment]
You mean regarding proxies from webshare? Follow my guide and afaik then on the order page you can reduce the amount down to 20.

**Reply by newworld00** [reply]
I want to do a one-month trial and don't have any problems with the program.thx

**Reply by jlplenio** [reply]
Sounds good. Don't forget to cancel your subscription immediately after ordering. Have fun.

**Reply by newworld00** [reply]
So what plan do you recommend, the private one and something else?

**Reply by jlplenio** [reply]
Follow this and reduce to 20 proxies:
https://github.com/jlplenio/crude-twitch-viewer-bot/wiki/Webshare.io-Proxies-Guide#--minimum-not-all-viewers-will-count-10month
